src/pybind/mgr/orchestrator.py

```diff
@@ -1162,7 +1195,7 @@ def __init__(self, name, placement=None, count=None):
         self.name = name
 
         #: Count of service instances
-        self.count = 1 if count is None else count
+        self.count = self.placement.count if self.placement is not None else 1  # for backwards-compatibility
```

Comment:
I'm working on making sure all orchestrators are validated with mypy in order to make it possible to refactor this.